news/2018-01-18-community-choice-2017.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Community Choice 2017"
+date: 2018-01-18 07:00:00 +0000
+---
+
+With 2017 well behind us, it's time to take a moment to sit back and reflect on which maps were truly the greatest of the year. Have your say and help us figure out which maps from 2017 are truly the Community's Favourite!
+
+[![](/wiki/shared/news/2018-01-18-community-choice-2017/banner.jpg)](https://osu.ppy.sh/community/contests/59)
+
+You may know them as the "best of" votes held historically throughout the past few years, but the votes are back for 2018 with a brand new look, and a better name!
+
+Each osu! player gets **10 votes per gamemode** to cast for their favourite ranked maps of 2017 that they've played. The voting is super simple - find your favourite 10 maps in any mode, shoot them a star, and you're done!
+
+The highest voted map for each gamemode will be declared the Community Favourite of 2017 and earn its mapper **a shiny new profile badge, and 6 months of osu!supporter** for their efforts.
+
+The top 10 maps for each mode will also grant their mappers **1 month of osu!supporter** as well.
+
+How do you vote? Easy! Head to the [contest listing](https://osu.ppy.sh/community/contests) and click the *Community Choice* panels, or follow the links below:
+
+* [osu!](https://osu.ppy.sh/community/contests/59)
+* [osu!taiko](https://osu.ppy.sh/community/contests/60)
+* [osu!catch](https://osu.ppy.sh/community/contests/61)
+* [osu!mania](https://osu.ppy.sh/community/contests/62)
+
+The voting will run for **14 days** from the date of this post, so get cracking, and make sure to tell all your friends!
+
+We'll also be hosting a similar poll purely for Loved maps at a later date - this is why no Loved maps are included in the entries for this vote. Stay tuned!
+
+Which map do you think will win?
```

Comment:
"maps" plural for covering all game modes? I personally also like a little reiteration of "The decision is yours!" or some similar empowering thing here, but maybe that's just me being corny.

Reply:
Which beatmap